runtime/libffi/z/sysvz64.s

```diff
@@ -40,19 +46,44 @@ FFISYS CELQPRLG DSASIZE=DSASZ,PSECT=ASP
 *         SR  6,6              Offset in stored parm values
           LA  6,0
 
+*Dumb handling for now, but if struct return with size > 24
```

Comment:
Please clean-up this comment here (`Dumb handling for now` ) as it seems the right logical step that allocates a pointer to R1 and makes GPR1 unavailable and if you think there could be a better way to do this, may be we can put `TODO`.

Reply:
This was an outdated todo, already handled with the current method